src/Particle/RealSpacePositions.h

```diff
@@ -43,6 +43,16 @@ class RealSpacePositions : public DynamicCoordinates
   }
   void setOneParticlePos(const PosType& pos, size_t iat) override { RSoA(iat) = pos; }
 
+  void mw_acceptParticlePos(const RefVector<DynamicCoordinates>& coords_list,
+                            size_t iat,
+                            const std::vector<PosType>& new_positions,
+                            const std::vector<bool>& isAccepted) override
+  {
+    for (size_t iw = 0; iw < isAccepted.size(); iw++)
+      if (isAccepted[iw])
```

Comment:
If setoneparticle pos is ever not cheap, would need an omp loop here. Is set() guaranteed to be cheap?

Reply:
Still exploring... for now it is very cheap.